### DISCUSSION TITLE
cURL error 35: OpenSSL SSL_connect

### DISCUSSION THREAD

**Discussion by youdou24** [tectalichq/public-openai-client-php#24] [Q&A] [answered]
cURL error 35: OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to api.openai.com:443 (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://api.openai.com/v1/chat/completions

**Comment by the-csaba** [comment, marked as answer]
H @youdou24 

Thank you for reporting this.
Please provide context and details about when and how you stumbled upon this error.
